src/gpio/erased.rs

```diff
@@ -0,0 +1,147 @@
+use super::*;
+
+pub type EPin<MODE> = ErasedPin<MODE>;
```

Comment:
I don't think this type alias is needed. I think `ErasedPin` is short enough. At least this type alias should not be used in this code base to benefit readability.